src/deep_links/mod.rs

```diff
@@ -39,14 +39,15 @@ pub struct OpenPlayerLink {
 #[derive(Default, Serialize, Debug, PartialEq, Eq)]
 #[serde(rename_all = "camelCase")]
 pub struct ExternalPlayerLink {
-    pub href: Option<String>,
     pub download: Option<String>,
     pub streaming: Option<String>,
+    pub playlist: Option<String>,
```

Comment:
Please add a descriptive docs on the field, e.g.

```suggestion
    /// Playlist contains a m3u data URI
    pub playlist: Option<String>,
```